### ISSUE TITLE
Are training hyperparameters the same for all types of pretrained models with different backbones?

### ISSUE THREAD

**Issue by OPHoperHPO** [Karel911/TRACER#12]
Hi, 

Thanks for your incredible research.
I tried train network from scratch on dataset with objects and tried multiple pretrained backbones with removed edge generation.
Model with backbone 0 or 1 came to very high accuracy during training with your model-training params, such `channels`, `RFB_aggregated_channel`, `frequency_radius`, `denoise`  and other values in the config.py.  

I tried move to heavy 7 backbone, but accuracy couldn't went to desireble values with same model parameters.

So, i got question, are model parameters such `channels`, `RFB_aggregated_channel`, `frequency_radius`, `denoise` and `gamma` used the same as in config.py for training your model with 7 backbone?
